test/ttt_test.exs

```diff
@@ -18,20 +18,35 @@ defmodule TTTTest do
     input4 = ",,"
     input5 = "10"
     input6 = "0"
-    input7 = "5"
+    input7 = "9"
+    input8 = "@@@"
+    input9 = "哈"
+    input10 = "-1"
+    input11 = "   1   "
+    input12 = "\#{1+1}"
+    input13 = "6666666666666666666666666"
+    input14 = "\n"
+
     assert TTT.valid_input?(input1) == :true
```

Comment:
Why is `TTT.valid_input?` returning `:true` and `:false`? Why not literally just return the booleans true and false instead of atoms?